mmpose/models/backbones/utils/transformer.py

```diff
@@ -796,7 +796,8 @@ def gen_encoder_output_proposals(self, memory, memory_padding_mask,
                 torch.linspace(
                     0, H - 1, H, dtype=torch.float32, device=memory.device),
                 torch.linspace(
-                    0, W - 1, W, dtype=torch.float32, device=memory.device))
+                    0, W - 1, W, dtype=torch.float32, device=memory.device),
+                indexing='ij')
```

Comment:
This function is torch.linspace instead of torch.meshgrid. Is this change intended?

Reply:
If you expand the context you can see it is a call to `torch.meshgrid` with two calls to `torch.linspace` as arguments.
```
            grid_y, grid_x = torch.meshgrid(
                torch.linspace(
                    0, H - 1, H, dtype=torch.float32, device=memory.device),
                torch.linspace(
-                    0, W - 1, W, dtype=torch.float32, device=memory.device))
+                    0, W - 1, W, dtype=torch.float32, device=memory.device),
                indexing='ij')
```

Reply:
Aha, my fault.